### DISCUSSION TITLE
Discrepancy between API and SellerCentral reports

### DISCUSSION THREAD

**Discussion by jebm** [amzn/selling-partner-api-models#3283] [General]
I am currently in the process of determining a way to roughly estimate a seller's annual FBA revenue. So far my approach is to programmatically download a years worth of data in one month increments as a .csv, merge them into one .csv, then total the gross value of all the rows. I chose to use the GET_FBA_FULFILLMENT_CUSTOMER_SHIPMENT_SALES_DATA report type since as per Amazon it "contains condensed item level data on shipped FBA customer orders including price, quantity, and ship to location". This seems to produce believable dollar values. I wanted to determine the accuracy of my approximations so I started comparing them to the Transaction Reports provided by Seller Central. From my understanding, these Transaction Reports contain the data of every transaction that occurred for a given seller's FBA account within the specified timeframe. By totaling the ProductSales column in the .csv downloaded from the Transaction Report page using the same timespan as the the API report, this should result in a value similar to the CUSTOMER_SHIPMENT_SALES_DATA report (ie. a gross revenue - no deductions/fees/taxes/refunds subtracted). For the first few tests I ran the numbers were very alike. For others, they were nearly half the value. I have included some of the results below.

Seller A
SellerCentral - $6,236,276 
API - $6,233,428

Seller B
SellerCentral - $541,794
API - $541,553

Seller C
SellerCentral - $6,535,788
API - $6,225,032

Seller D
SellerCentral - $11,711,879
API - $6,346,711

Seller E
SellerCentral - $3,475,356
API - $2,988,183

### Summarizing questions
Is there a source of truth provided by AMZ when it comes to determining the annual revenue of a FBA Seller?
Am I incorrect in assuming the Transaction Reports and GET_FBA_FULFILLMENT_CUSTOMER_SHIPMENT_SALES_DATA Reports would use similar/the same data?
Is there some other explanation for the discrepancy on Seller D or E?

**Comment by ANarayanan-Cervello** [comment]
Hi @jebm ,did you try raising a support case with the Amazon Seller/Vendor Central team? If yes ,what was their response? 
I am too, seeing discrepancies between what the API has to offer versus what's available on the portal.

**Reply by jebm** [reply]
I never escalated this with Amazon. It turns out I was incorrect in assuming the Transaction Reports and GET_FBA_FULFILLMENT_CUSTOMER_SHIPMENT_SALES_DATA Reports would use the same data. I'm still unsure why there is such a large gap between some of the values when using these 2 reports since you would think the datasets would be similar. I discovered that Seller Central offers a Customer Shipment Sales report that matched the numbers I was getting from the API and this was my solution. Sorry if this doesn't help your situation.

**Comment by therubioftw** [comment]
Hi @jebm - I found that the SellerCentral uses the Seattle timezone, while in the API they're using UTC.
Once we fixed that, we got the $$$ to match down to the cents...

**Reply by NetRushJacob** [reply]
All North America (CA/US/MX) SellerCentral portals use Seattle time zone (PST/PDT).
All EU SellerCentral portals use CET/CEST

All reports and APIs use UTC
